attr_default.gemspec

```diff
@@ -7,6 +7,8 @@ Gem::Specification.new do |gem|
   gem.summary       = %q{Dynamic Ruby defaults for ActiveRecord attributes. These are lazy evaluated just in time: when first accessed, or just before validation or save. This allows dynamic defaults to depend on attributes that are assigned after initialization, or on other dynamic defaults.}
   gem.homepage      = "https://github.com/RingRevenue/attr_default"
 
+  gem.metadata['allowed_push_host'] = "https://gem.fury.io/invoca"
```

Comment:
This is a public repo, so it should push to RubyGems, right? It is there now, I believe.